File: PatasSolidarias.Frontend/src/api/usuario-response.ts
import { EUsuarioTipo } from './e-usuario-tipo';

export class UsuarioResponse {
  id?: number;
  nome?: string;
  email?: string;
  criadoDataHora?: Date;
  usuarioTipoId?: EUsuarioTipo;
  descricao?: string;
}

File: PatasSolidarias.Api/Models/UsuarioResponse.cs
using PatasSolidarias.Domain.Entities;

namespace PatasSolidarias.Backend.Models;

public class UsuarioResponse
{
    public int Id { get; set; }
    public required string Email { get; set; }
    public required string Nome { get; set; }
    public DateTime CriadoDataHora { get; set; }
    public EUsuarioTipo UsuarioTipoId { get; set; }
    public string? Descricao { get; set; }
}